sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExpressions.scala

```diff
@@ -282,7 +282,9 @@ object CudfBinaryExpression {
     BinaryOp.LESS -> ast.BinaryOperator.LESS,
     BinaryOp.LESS_EQUAL -> ast.BinaryOperator.LESS_EQUAL,
     BinaryOp.LOGICAL_AND -> ast.BinaryOperator.NULL_LOGICAL_AND,
+    BinaryOp.NULL_LOGICAL_AND -> ast.BinaryOperator.NULL_LOGICAL_AND,
```

Comment:
Nit: It would be nice if this was kept in lexicographical order as it was before.